app/src/main/java/com/jerboa/Utils.kt

```diff
@@ -374,6 +374,32 @@ fun LazyListState.isScrolledToEnd(): Boolean {
     return out
 }
 
+/**
+ * Open a sharesheet for the given post.
+ * If the post has a URL, share that.
+ * Otherwise share the post directly.
+ */
+fun sharePost(post: Post, ctx: Context) {
+    post.url?.let {
```

Comment:
Perhaps we could break this out into two share actions so the user can choose "Share link" or "Share post"? That's how Boost and Bacon Reader work.

Reply:
Sounds good. I wasn't sure what the feeling was on multiple share links.

Reply:
Added both Share Link and Share Post.